### PR TITLE
Fix database migration versions table

### PR DESCRIPTION
**Pull request by matthauck**

### CODE REVIEW

File: src/db_migrations.rs [hunk 1]
```diff
@@ -80,3 +82,37 @@ pub fn migrate(conn: &mut Connection) -> Result<()> {
...
 }
+
+#[cfg(test)]
+mod tests {
+    extern crate tempdir;
+
+    use self::tempdir::TempDir;
+    use super::*;
+
+    #[test]
```
Comment (matthauck) [inline src/db_migrations.rs]: Note: these tests reproduced the problem I was seeing when I tried to deploy the latest octobot. :)

**Comment by briangreenery** [comment]
What was the error?

**Comment by briangreenery** [comment]
Oh duplicate rows

**Comment by briangreenery** [comment]
Wait, was that the problem?

**Comment by matthauck** [comment]
> ---- db_migrations::tests::test_migration_versions stdout ----
	thread 'db_migrations::tests::test_migration_versions' panicked at 'assertion failed: `(left == right)`
  left: `Some(1)`,
 right: `Some(0)`', src/db_migrations.rs:99:8
>
> ---- db_migrations::tests::test_multiple_migration stdout ----
	thread 'db_migrations::tests::test_multiple_migration' panicked at 'assertion failed: migrate(&mut conn).is_ok()', src/db_migrations.rs:109:8


Yeah, I kinda expected `REPLACE` to work... but it wasn't doing what I expected...

**Comment by matthauck** [comment]
There was basically nothing to trigger a conflict that would result in a replace. Could have also added an "id" primary key column or something that was always "1".  /shrug

**Pull request merged**